src/components/10-atoms/textarea/index.js

```diff
@@ -92,35 +104,33 @@ class AXATextarea extends NoShadowDOM {
 
   get charsLeft() {
     const {
-      maxLength,
+      maxLength = 0,
       nativeInput: { value: nativeValue },
     } = this;
 
     if (nativeValue) {
-      return maxLength - nativeValue.length - 1;
+      return maxLength - nativeValue.length;
```

Comment:
When setting `maxLength` to 1, it shows that I have 0 chars left. Is this meant to be?
<img width="978" alt="Screenshot 2019-10-08 at 08 32 55" src="https://user-images.githubusercontent.com/38284563/66372457-54ea6080-e9a6-11e9-8a32-d50dad61998f.png">

Reply:
This was decided. It's also like that on prod: http://patterns.axa.ch/iframe.html?id=atoms-textarea--textarea&knob-label*=&knob-name*=&knob-refId=&knob-placeholder=&knob-error=&knob-counter=&knob-countermax=&knob-maxlength=1